packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -663,10 +663,17 @@ export interface CloudMapOptions {
    */
   readonly name?: string,
 
+  /**
+   * The service discovery namespace created in this service.
```

Comment:
But it's not "created in this service", is it?

It's an existing SD namespace this service will register itself into.

Reply:
Addressed